filetracker/__init__.py

```diff
@@ -220,6 +232,17 @@ class DataStore(object):
     """An abstract base class giving access to storing and retrieving files'
        content."""
 
+    FileInfoEntry = collections.namedtuple(
+        'FileInfoEntry', ['name', 'mtime', 'size'])
+    """Information entry for single file.
+
+        Fields:
+
+        * ``name`` unversioned name of given file
```

Comment:
Czy dobrze rozumiem, że ponieważ wersja nie jest zwracana przez list_files(), to cache cleaner nie ma możliwości ustrzec się przed poniższym wyścigiem?

1. Cache cleaner buduje indeks, wywołując list_files().
2. Ktoś zapisuje nowszą wersję pliku, który dawno nie był modyfikowany.
3. Cache cleaner na podstawie wcześniej zbudowanego indeksu usuwa pliki. Ponieważ usuwa po nazwie niewersjonowanej, to usunie nowszą wersję.

Dobrze mi się wydaje? Jeśli tak, to trzeba zmienić logikę tak, żeby odwoływać się do plików po pełnych nazwach z wersją przy usuwaniu oraz jakoś obsłużyć (np. nic nie robiąc) sytuację, gdy plik już nie istnieje.